sdk_lib/make_chroot.sh

```diff
@@ -154,6 +159,13 @@ init_setup () {
    ln -sf "${CHROOT_TRUNK_DIR}/src/third_party/portage-stable" \
      "${FLAGS_chroot}"/"${PORTAGE_STABLE_OVERLAY}"
 
+   if [[ -n "${FLAGS_extra_portdir_overlay_path}" ]]; then
+     if [[ -n "${FLAGS_extra_portdir_overlay_name}" ]]; then
+       ln -sf "${CHROOT_TRUNK_DIR}"/"${FLAGS_extra_portdir_overlay_path}" \
```

Comment:
Just checking I follow you here, are you suggesting to set the raw path e.g. ```/mnt/host/.../overlay-name``` directly to the ```PORTDIR_OVERLAY``` variable to avoid the need for the symlink?